AssociateAPI/src/main/java/com/bae/associate/AssociateImpl.java

```diff
@@ -0,0 +1,47 @@
+package com.bae.associate;
```

Comment:
Just to check, should this be in a sub-package? Or should be at the same level as main?